server/repositories/offender.js

```diff
@@ -75,17 +80,15 @@ function offenderRepository(prisonApiHttpClient, incentivesApiHttpClient) {
   }
 
   function getAdjudicationsFor(offenderNo) {
-    return prisonApiHttpClient.get(
-      `${prisonApiBaseUrl}/offenders/${offenderNo}/adjudications`,
-      {
-        'Page-Limit': config.prisonApi.adjudications.pageLimit,
-      },
-    );
+    const endpoint = `${adjudicationsApiBaseUrl}/adjudications/${offenderNo}/adjudications`;
```

Comment:
[Here is this endpoint](https://github.com/ministryofjustice/hmpps-manage-adjudications-api/blob/9e4fb75fe356845eda7c96c66aab998f792aecee/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/AdjudicationSummaryController.kt#L142-L208) in the AdjudicationsAPI